src/configuration/services/rabbitmq.md

```diff
@@ -34,7 +34,7 @@ In your ``.platform/services.yaml``:
 ```yaml
 myrabbitmq:
     type: rabbitmq:3.5
```

Comment:
We should also update this to default to RabbitMQ 3.7 in the example, not 3.5. @xtfer